app/views/your-psp/_worldpay.njk

```diff
@@ -2,32 +2,35 @@
 
 <p class="govuk-body">Go to your <a class="govuk-link" href="https://secure.worldpay.com/sso/public/auth/login.html">Worldpay account</a> to get the details you need to enter here, or <a class="govuk-link" href="https://docs.payments.service.gov.uk/switching_to_live/set_up_a_live_worldpay_account/#connect-your-live-account-to-worldpay">read more in our documentation.</a></p>
 
-<h2 class="govuk-heading-m govuk-!-margin-top-9">Account credentials</h2>
-
 {% set isAccountCredentialsConfigured = (credential.credentials and credential.credentials.one_off_customer_initiated|length) %}
 
 {{
   govukSummaryList({
+    card: {
+      title: {
+        text: "Account credentials"
+      },
+      actions: {
+        items: [
+          {
+            href: formatAccountPathsFor(routes.account.yourPsp.credentialsWithGatewayCheck, currentGatewayAccount.external_id, credential.external_id),
+            text: "Change",
+            visuallyHiddenText: "account credentials",
+            attributes: {
+              id: "credentials-change-link"
+            }
+          }
+        ]
+      }
+    },
     rows: [
       {
         key: {
           text: "Merchant code"
         },
         value: {
-          text: credential.credentials.one_off_customer_initiated.merchant_code if isAccountCredentialsConfigured else "Not configured",
+        text: credential.credentials.one_off_customer_initiated.merchant_code if isAccountCredentialsConfigured else "Not configured",
```

Comment:
I think the indentation got messed up slightly by conflict resolution. Can be fixed in next PR.

Reply:
Thanks - will fix this in the next PR.